package.json

```diff
@@ -41,9 +41,6 @@
     "@react-stately/radio": "^3.8.1",
     "@react-stately/tabs": "^3.4.1",
     "@react-stately/tree": "^3.6.1",
-    "@react-types/menu": "^3.9.1",
-    "@react-types/numberfield": "^3.4.2",
-    "@react-types/radio": "^3.4.2",
```

Comment:
These packages are used by `@react-aria/menu`, `@react-aria/numberfield`,… so there's no change to `yarn.lock`